translations/binaries/slps_strings_items_weapon.json

```
@@ -0,0 +1,1299 @@
+        },
+        {
+            "offset": 1778880,
+            "string": "[Sign_Sword]ストライクアックス[END]",
+            "translation": "[Sign_Sword]스트라이크 액스[END]",
+            "original_bytes_length": 24,
+            "available_bytes_size": 24
+        },
+        {
+            "offset": 1778816,
+            "string": "対剣士用に作られた斧。[LINE]\n刃の隙間は剣を引っ掛け[LINE]\n折るためのもの。[END]",
+            "translation": "대검사용으로 만들어진 도끼.[LINE]\n날의 틈새는 검을 걸어[LINE]\n부러뜨리기 위한 것이다.[END]",
+            "original_bytes_length": 63,
+            "available_bytes_size": 63
+        },
+        {
+            "offset": 1778792,
+            "string": "[Sign_Sword]ドライビンアックス[END]",
+            "translation": "[Sign_Sword]드라이빈 액스[END]",
+            "original_bytes_length": 24,
+            "available_bytes_size": 24
+        },
+        {
+            "offset": 1778704,
+            "string": "特徴的な薄く丸い刃を持つ斧。[LINE]\n力のない者が振るっても[LINE]\n容易く肉を裂けられるほど鋭い。[END]",
+            "translation": "특징적인 얇고 둥근 칼날을 가진 도끼.[LINE]\n힘이 없는 자가 휘두르더라도[LINE]\n쉽게 고기를 찢을 정도로 날카롭다.[END]",
+            "original_bytes_length": 83,
+            "available_bytes_size": 87
+        },
+        {
+            "offset": 1778680,
+            "string": "[Sign_Sword]レアアックス[END]",
+            "translation": "[Sign_Sword]레어 액스[END]",
+            "original_bytes_length": 18,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1778592,
+            "string": "名工ギースの魂がこもった[LINE]\n無二の傑作。きわめて希少価値が[LINE]\n高く、戦士たちの憧れの斧。[END]",
+            "translation": "명공 기스의 영혼이 담긴[LINE]\n무이의 걸작. 매우 희귀가치가[LINE]\n높아, 전사들의 동경의 도끼.[END]",
+            "original_bytes_length": 83,
+            "available_bytes_size": 87
+        },
+        {
+            "offset": 1778568,
+            "string": "[Sign_Sword]トマホークアックス[END]",
+            "translation": "[Sign_Sword]토마호크 액스[END]",
+            "original_bytes_length": 24,
+            "available_bytes_size": 24
+        },
+        {
+            "offset": 1778488,
+            "string": "「切るための道具」[LINE]\nという意味の名を持つ斧。[LINE]\n扱いが上手ければ投げることも可能。[END]",
+            "translation": "「자르기 위한 도구」[LINE]\n라는 의미의 이름을 가진 도끼.[LINE]\n다루기가 능숙하다면 던지기도 가능하다.[END]",
+            "original_bytes_length": 79,
+            "available_bytes_size": 79
+        },
+        {
+            "offset": 1778464,
+            "string": "[Sign_Sword]ポールアックス[END]",
+            "translation": "[Sign_Sword]폴 액스[END]",
+            "original_bytes_length": 20,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1778400,
+            "string": "非常に重く、鋭い切っ先を持つ斧。[LINE]\n重装備の鎧でさえ打ち砕ける。[LINE]\n[END]",
+            "translation": "매우 무겁고, 날카로운 칼날을 가진 도끼.[LINE]\n중장비 갑옷조차 부술 수 있다.[LINE]\n[END]",
+            "original_bytes_length": 63,
+            "available_bytes_size": 63
+        },
+        {
+            "offset": 1778368,
+            "string": "[Sign_Sword]ジャッドバラアックス[END]",
+            "translation": "[Sign_Sword]잭 바라 액스[END]",
```

```suggestion
            "translation": "[Sign_Sword]제드버러 액스[END]",
```

@Da1eth 님의 정보를 토대로 원래의 이름이 Jedburgh axe를 일본식으로 적었다고 추정했을 때, Jedburgh는 'JED-bur-uh'라고 읽기 때문에 '제드버러'로 표기합니다.
J가 묵음일 수 있는지 검색해 보았지만 적절한 자료를 찾지 못했습니다.